native/package.json

```diff
@@ -20,11 +20,13 @@
     "android:reinit": "adb shell pm clear com.integreat && adb shell am start com.integreat/com.integreat.MainActivity",
     "android:start-avd": "bash -c '$ANDROID_HOME/emulator/emulator -avd $($ANDROID_HOME/emulator/emulator -list-avds | head -n 1)'",
     "test": "cross-env BUILD_CONFIG_NAME='integreat-test-cms' jest --config ./jest.config.js",
-    "test:coverage": "cross-env BUILD_CONFIG_NAME='integreat-test-cms' jest --config ./jest.config.js --coverage",
+    "test:coverage": "cross-env BUILD_CONFIG_NAME='integreat-test-cms' jest --config ./jest.config.js --coverage",   
+    "test:ci": "cross-env BUILD_CONFIG_NAME='integreat-test-cms' jest --config ./jest.config.js --coverage --ci --testResultsProcessor=jest-junit",
     "test:watch": "cross-env BUILD_CONFIG_NAME='integreat-test-cms' jest --config ./jest.config.js --watchAll",
     "test:update": "cross-env BUILD_CONFIG_NAME='integreat-test-cms' jest --config ./jest.config.js -u",
     "test:e2e": "cross-env BUILD_CONFIG_NAME='integreat-e2e' jest --config jest.e2e.config.js --maxWorkers=4",
-    "lint": "eslint .",
+    "lint": "yarn run eslint",
+    "eslint": "eslint .",
```

Comment:
Why do we need two commands for this?

Reply:
Because I want to have the "eslint" command available in every script. In the webapp "lint" executes eslint and stylelint.